### DISCUSSION TITLE
[FEATURE]: tag 里面的字段还不支持使用 processor 插件处理

### DISCUSSION THREAD

**Discussion by wangzan18** [alibaba/ilogtail#792] [Help] [answered]
**Concisely describe the proposed feature**
1、输出到 kafka 的数据
"tags": {
    "k8s.namespace.name":"java_app",
    "host.ip": "192.168.6.128",
    "host.name": "master",
    "log.file.path": "/data/test.log"
  },

**Describe the solution you'd like (if any)**
2、我想通过 processor 的字段删除功能删除某些 tag
processor_drop processor插件可以丢弃日志字段。
- Type: processor_drop
    DropKeys: 
      - key1

**Comment by messixukejia** [comment]
欢迎一起进来参与开发，需要支持可以群里@下我们

**Comment by wangzan18** [comment]
群里的大佬让我提交一个 issue 的，我是小菜鸟

**Comment by messixukejia** [comment, marked as answer]
https://github.com/alibaba/ilogtail/blob/main/docs/cn/data-pipeline/flusher/flusher-kafka_v2.md

使用TagFieldsRename，把vaule字段设置成空

**Reply by wangzan18** [reply]
我测试了一下，不行，数据到 kafka 之后还是原样

我设置的是

    Convert:
      TagFieldsRename:
        tags.container.image.name:

**Reply by messixukejia** [reply]
https://github.com/alibaba/ilogtail/blob/main/docs/cn/developer-guide/log-protocol/converter.md
字段看下这个。

麻烦把这个场景在[flusher-kafka_v2.md](https://github.com/alibaba/ilogtail/blob/main/docs/cn/data-pipeline/flusher/flusher-kafka_v2.md)里提交pr完善下吧

**Reply by wangzan18** [reply]
我来提交 PR 么，我不太懂，我只是一个使用者而已

**Reply by messixukejia** [reply]
没关系的，问题解决了，非常欢迎使用者参与到ilogtail社区。一些文档还是需要大家一起来完善的

**Reply by shalousun** [reply]
删除tag数据的存储使用场景可以描述下，从我过去的日志分析检索经验看这几个字段存起来也是有必要的，纯业务场景可能日志文件路径存储要求倒是必要性不高。只要可观测性的展示做的好，这几个字段基本可以用上

**Reply by wangzan18** [reply]
也没有特定的场景，一些字段基本上用不上，就不想让他打入到 kafka 了，也节省存储空间，不过目前不影响我的使用，非常感谢徐可甲老师的耐心解答，我打入 ES 的时候使用 logstash 处理掉了

**Comment by llthmos** [comment]
TagFieldsRename 这个我看文档是把tag的key重新命名, 如何把value用空字符串替换呢

**Reply by messixukejia** [reply]
![image](https://github.com/alibaba/ilogtail/assets/13250519/142e132b-2e3e-469a-85ca-b0ff5e0ab180)
